src/app/atoms/Money.tsx

```diff
@@ -106,6 +108,19 @@ interface MoneyWithoutFormatProps {
   smallFractionFont: boolean;
 }
 
+const formatAmount = (bn: BigNumber) => {
+  const gte1000 = bn.gte(1000);
+
+  if (!gte1000) return { amount: bn, gte1000 };
+
+  const n = Number(bn);
+  const calc = Math.floor(n * 100) / 100;
+
+  const amount = new BigNumber(calc.toFixed(2));
+
+  return { amount, gte1000 };
+};
+
```

Comment:
```suggestion
const FORMATTING_THRESHOLD = 1000;
export const formatAmount = (amount: BigNumber) => {
  const isLessThanThreshold = amount.isLessThanOrEqualTo(FORMATTING_THRESHOLD);

  if (isLessThanThreshold) return { amount, isLessThanThreshold };

  const n = Number(amount);
  const calc = Math.floor(n * 100) / 100;

  const amountFormatted = new BigNumber(calc.toFixed(2));

  return { amountFormatted, isLessThanThreshold };
};

```

Reply:
Fix naming, please. What does `n` and `calc` mean?
Fix magic numbers too, please. What does `100` mean?